www/universal_links.js

```diff
@@ -79,14 +79,14 @@ var universalLinks = {
    *
    * @param {number} milliseconds - Optional. The number of milliseconds to wait before executing the code. If omitted, the value 0 is used
    */
-  checkDeepLink: function (milliseconds = 2000) {
+  checkDeepLink: function () {
```

Comment:
Missing milliseconds parameter.

Reply:
See my comment below to the other comment. :)